File: C/src/1006.c
#include <stdio.h>

int main(){
    float x,y,z;
    scanf("%f\n%f\n%f",&x,&y,&z);
    float media = (2*x+3*y+5*z)/10;
    printf("MEDIA = %.1f\n",media);
    return 0;
}

File: go/1006.go
package main

import "fmt"

func main() {
	var x, y, z float64
	fmt.Scan(&x)
	fmt.Scan(&y)
	fmt.Scan(&z)
	var average float64 = (2*x + 3*y + 5*z) / 10
	fmt.Printf("MEDIA = %.1f\n", average)
}
